reference/CMakeLists.txt

```diff
@@ -1,4 +1,6 @@
 add_library(ginkgo_reference $<TARGET_OBJECTS:ginkgo_reference_device> "")
+list(APPEND GINKGO_WINDOWS_PATH ${CMAKE_CURRENT_BINARY_DIR})
+set(GINKGO_WINDOWS_PATH ${GINKGO_WINDOWS_PATH} PARENT_SCOPE)
```

Comment:
Since these two lines are basically everywhere, I would prefer them in a single macro named what it does (not sure if it is accurate, but for example: `prepare_windows_path`). Just from these two lines, I have no Idea why we would need it.
Additionally, when we add another function / macro, these two lines are very much error prone because a simple typo in any of the variables would not result in an error, but in the wrong behavior.